addons/mass_mailing/migrations/13.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,227 @@
+mass_mailing / mail.mass_mailing.contact / list_ids (many2many)          : table is now 'mailing_contact_list_rel' ('mail_mass_mailing_contact_list_rel')
+mass_mailing / mail.mass_mailing.contact / tag_ids (many2many)           : table is now 'mailing_contact_res_partner_category_rel' ('mail_mass_mailing_contact_res_partner_category_rel')
+# DONE: pre-migration renamed table
+mass_mailing / mail.mass_mailing.contact / subscription_list_ids (one2many): relation is now 'mailing.contact.subscription' ('mail.mass_mailing.list_contact_rel') [nothing to do]
+mass_mailing / mail.mass_mailing.contact / website_message_ids (one2many): module is now 'portal' ('mass_mailing')
+# NOTHING TO DO
+mass_mailing / mail.mass_mailing.list   / contact_ids (many2many)       : relation is now 'mailing.contact' ('mail.mass_mailing.contact') [nothing to do]
+mass_mailing / mail.mass_mailing.list   / contact_ids (many2many)       : table is now 'mailing_contact_list_rel' ('mail_mass_mailing_contact_list_rel')
+# DONE: pre-migration renamed table
+mass_mailing / mail.mass_mailing.list   / subscription_contact_ids (one2many): DEL relation: mail.mass_mailing.list_contact_rel
+mass_mailing / mail.mass_mailing.list_contact_rel / contact_id (many2one)         : relation is now 'mailing.contact' ('mail.mass_mailing.contact') [nothing to do]
+mass_mailing / mail.mass_mailing.list_contact_rel / list_id (many2one)            : relation is now 'mailing.list' ('mail.mass_mailing.list') [nothing to do]
+# NOTHING TO DO
+mass_mailing / mail.mass_mailing.stage  / __last_update (datetime)      : module is now 'utm' ('mass_mailing')
+mass_mailing / mail.mass_mailing.stage  / display_name (char)           : module is now 'utm' ('mass_mailing')
+mass_mailing / mail.mass_mailing.stage  / name (char)                   : module is now 'utm' ('mass_mailing')
+mass_mailing / mail.mass_mailing.stage  / sequence (integer)            : module is now 'utm' ('mass_mailing')
+mass_mailing / mail.mass_mailing.tag    / __last_update (datetime)      : module is now 'utm' ('mass_mailing')
+mass_mailing / mail.mass_mailing.tag    / color (integer)               : module is now 'utm' ('mass_mailing')
+mass_mailing / mail.mass_mailing.tag    / display_name (char)           : module is now 'utm' ('mass_mailing')
+mass_mailing / mail.mass_mailing.tag    / name (char)                   : module is now 'utm' ('mass_mailing')
+# DONE: Moved to utm in utm module
+mass_mailing / mailing.contact          / email_normalized (char)       : NEW isfunction: function, stored
+mass_mailing / mailing.list             / subscription_ids (one2many)   : NEW relation: mailing.contact.subscription
+# NOTHING TO DO
+mass_mailing / mail.mass_mailing        / statistics_ids (one2many)     : DEL relation: mail.mail.statistics
+mass_mailing / mailing.mailing          / mailing_trace_ids (one2many)  : NEW relation: mailing.trace
+# NOTHING TO DO: one2many
```

Comment:
Done: in premigration: rename field

Reply:
one2many fields not saving any data. they are like a computed field. am i right?

Reply:
Yeah, but you have record in ir.model.field

Reply:
In order to avoid creating a field by ORM use to rename in SQL